.github/workflows/timeline-check.yml

```diff
@@ -3,6 +3,9 @@ name: timeline-check
 on:
   pull_request:
     types: [labeled, unlabeled, opened, synchronize, reopened]
+    paths-ignore:
+      - 'docs/**'
+      - '.github/**'
```

Comment:
Like default.yml, shouldn't we also add:
`- '!.github/workflows/timeline-check.yml'`
for the cases when it is self-modified?

Reply:
And how about other workflows?